src/components/tabs/README.md

```diff
@@ -92,7 +183,7 @@ In some situations, you may want to add classes to the `<li>` (nav-item) and/or
 the `title-item-class` prop (for the `<li>` element) or `title-link-class` prop (for the
 `<a>` element).  Value's can be passed as a string or array of strings.
 
-Note: THe `ative` class is automatically applied to the `<a>` element. You may need to accomodate
+Note: THe `ative` class is automatically applied to the active tabs `<a>` element. You may need to accomodate
```

Comment:
Two typos: `THe` and `ative`.

Reply:
Thanks! 👍

Reply:
No problem! Just happened to notice while looking through the changes. 😄